processor/src/main/java/org/mapstruct/ap/internal/util/accessor/AbstractAccessor.java

```diff
@@ -32,9 +35,20 @@
 abstract class AbstractAccessor<T extends Element> implements Accessor {
 
     protected final T element;
+    private final TypeMirror builderType;
 
     AbstractAccessor(T element) {
         this.element = element;
+        if (element.getEnclosingElement().getKind() == ElementKind.CLASS) {
```

Comment:
Yeah.  I'm going to try to get this working without adding builderType to Accessor.  Should I just update this PR if I make changes?  Or wait for other's feedback?